apps/render_cf.py

```diff
@@ -11,35 +12,245 @@ def snake_to_pascal_case(input_string: str):
     return ''.join([i.title() for i in split_string])
 
 
-def render_templates(job_types, security_environment, api_name):
+def get_states_for_jobs(job_types: dict) -> dict:
+    states = {}
```

Comment:
Now that there is significantly more functionality here, it's probably worth some basic docstrings. If you don't feel like that can be done quickly enough, don't worry about it. I'd rather get to testing.

Reply:
Might just be rewritten with CDK at some point, so not going to bother with documentation.